spring-cloud-dataflow-docs/src/main/asciidoc/configuration-local.adoc

```diff
@@ -62,6 +62,21 @@ java -jar spring-cloud-dataflow-server/target/spring-cloud-dataflow-server-{proj
     --spring.datasource.driver-class-name=org.mariadb.jdbc.Driver &
 ----
 
+The following example shows how to define a MariaDB database connection with command Line arguments
```

Comment:
Perhaps: 

> The following example shows how to configure MariaDB database connection with command-line arguments

Reply:
I copy pasted other sections and just changed db name, so keeping it consistent. Not gonna change.